content/en/blog/_posts/2023-10-31-Gateway-API-GA/index.md

```diff
@@ -60,7 +60,7 @@ follow up blog post that will cover each of these new features in detail.
 ### Everything else
 For a full list of the changes included in this release, please refer to the
 [v1.0.0 release
-notes](https://github.com/kubernetes-sigs/gateway-api/releases/tag/v0.1.0).
+notes](https://github.com/kubernetes-sigs/gateway-api/releases/tag/v1.0.0 ).
```

Comment:
There is a trailing whitespace that can be deleted.
```suggestion
notes](https://github.com/kubernetes-sigs/gateway-api/releases/tag/v1.0.0).
```

Reply:
oops my mistake! Thanks for review :)